__mocks__/translation.mock.js

```diff
@@ -5,31 +5,30 @@ const Localize = ({ i18n_default_text, components = [], values = {} }) => {
     const parts = i18n_default_text.split(/(<\d+>.*?<\/\d+>|{{\w+}})/g);
 
     const replaceValues = text => {
-        return text.replace(/{{(\w+)}}/g, (match, key) => values[key] || match);
+        return text.replace(/{{(\w+)}}/g, (match, key) => {
+            // If the value is an empty string, return an empty fragment to render nothing
+            if (values[key] === '') {
+                return '';
+            }
```

Comment:
Use case example for empty string value in `MT5AccountAdded`:
<img width="545" alt="image" src="https://github.com/user-attachments/assets/ffd62c97-53d2-409e-8cb3-f646af7a7d14">